interface/super/edit_layout.php

```diff
@@ -1350,7 +1350,7 @@ function myChangeCheck() {
 <input type="hidden" name="selectedfields" id="selectedfields" value="" />
 <input type="hidden" id="targetgroup" name="targetgroup" value="" />
 
-<div class="menubar" style='padding:5px 0;'>
+<div class="menuBar" style='padding:5px 0;'>
```

Comment:
This class menuBar can go away because it has a fixed color background.
I'd just do
```<div class="fixed-top py-2 px-1 bg-light text-dark">```
and remove the class in interface/themes/theme-defaults.scss

Reply:
k, will do

Reply:
proposed change looks good and will bring in after pass travis

Reply:
cool thanks. May have to hold patch as someone responded with billing issue. I'm looking over now!!

Reply:
bringing in :)